tasks/mule.yml

```
@@ -4,7 +4,7 @@
     group: root
     mode: 0644
     src: galaxy.service.j2
-    dest: /etc/systemd/system/galaxy.service
+    dest: /etc/systemd/system/{{ galaxy_systemd_sub_dest }}/galaxy.service
```

I'm not sure this works, to put them in folders like this, both are still called `galaxy.service` which is the important part for systemd AFAIK. We can instead leave them at the same level and just make the name configurable for the same result.

```suggestion
    dest: /etc/systemd/system/{{ galaxy_systemd_unit_name }}.service
```

---

i agree, i will edit the changes